rpc/dispatch.go

```diff
@@ -24,7 +24,7 @@ type dispatch struct {
 	log LogInterface
 }
 
-func newDispatch(enc encoder, calls *callContainer, l LogInterface) *dispatch {
+func newDispatch(enc *framedMsgpackEncoder, calls *callContainer, l LogInterface) *dispatch {
```

Comment:
Let's rename `dispatch` to `dispatcher`, and `packetHandler` to `packetizer`. It's clearer terminology anyway.

Reply:
Per our chat, renamed `packetHandler` only, since there's an existing `dispatcher` interface.